Softeq.XToolkit.WhiteLabel.Essentials.Droid/ImagePicker/ImagePickerActivityResultHandler.cs

```diff
@@ -0,0 +1,62 @@
+// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+using Android.Graphics;
+using Android.Media;
+using Android.Net;
+using Android.OS;
+using Softeq.XToolkit.Common.Weak;
+
+namespace Softeq.XToolkit.WhiteLabel.Essentials.Droid.ImagePicker
+{
+    public sealed class ImagePickerActivityResultHandler : IImagePickerActivityResultHandler
+    {
+        private WeakReferenceEx<Activity> _activityRef;
+
+        public void SetActivity(Activity activity)
+        {
+            _activityRef = new WeakReferenceEx<Activity>(activity);
```

Comment:
Seems like we need Activity reference here only for context. Can't we use ICurrentActivity as dependency for that?

Reply:
Currently, we use `IContextProvider->CurrentActivity`